jitbuilder/build/files/common.mk

```diff
@@ -174,7 +174,7 @@ JIT_PRODUCT_BACKEND_SOURCES+=\
     $(JIT_OMR_DIRTY_DIR)/optimizer/VPConstraint.cpp \
     $(JIT_OMR_DIRTY_DIR)/optimizer/VPHandlers.cpp \
     $(JIT_OMR_DIRTY_DIR)/optimizer/VPHandlersCommon.cpp \
-    $(JIT_OMR_DIRTY_DIR)/optimizer/ValuePropagation.cpp \
```

Comment:
This appears to be a duplicated set of the same lines above. Can you delete this set since we only need one?